user/getting-started.md

```diff
@@ -4,11 +4,7 @@ layout: en
 permalink: /user/getting-started/
 ---
 
-The very short guide to using Travis CI with your GitHub hosted code repository. Totally new to all this? Try reading [Travis CI for Complete Beginners](/user/for-beginners) instead.
-
-Or if you want a more complete guide to a particular language, pick one of these:
-
-{% include languages.html %}
+The very short guide to using Travis CI with your GitHub hosted code repository. Totally new to continuous integreation and Travis CI? Start with [Travis CI for Complete Beginners](/user/for-beginners) instead.
```

Comment:
Typo: `integration`

And since this paragraph is getting touched already, maybe we can omit the `Totally` as well. :-)